### PR TITLE
docs(CHANGELOG) changelog for 2.5.0-rc1

### PR DESCRIPTION
**Pull request by kikito**
This CHANGELOG was up to date with the PRs merged on 2021-06-28.

Compare link: https://github.com/Kong/kong/compare/2.4.1...master

Note that I have left several steps on git for easier reviews from the docs team. This pr should be squashed onto a single commit when merging.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -57,9 +58,135 @@
...
+- Bumped `grpcurl` from 1.8.0 to 1.8.1 [#7128](https://github.com/kong/kong/pull/7128)
+- Pinned `lua-protobuf` to 0.3.2 (previously unpinned) [#7079](https://github.com/kong/kong/pull/7079)
+
+### Additions
+
+#### Core
+
+- Control Planes can now send updates to new Data Planes even if the Control Planes lose connection to the database
+  [#6938](https://github.com/kong/kong/pull/6938)
+- Kong now automatically adds `cluster_cert`(`cluster_mtls-shared`) or `cluster_ca_cert`(`cluster-mtls=pki`) into
```
Comment (javierguerragiraldez) [inline CHANGELOG.md:91]:
```suggestion
- Kong now automatically adds `cluster_cert`(`cluster_mtls=shared`) or `cluster_ca_cert`(`cluster_mtls=pki`) into
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -57,9 +58,162 @@
...
+  [#7379](https://github.com/kong/kong/pull/7379)
+
+#### PDK
+
+- Accept tables in response body for stream subsystem
+  [#7082](https://github.com/kong/kong/pull/7082)
+
+#### Plugins
+
+- **hmac-auth**: add support for the "@request-target" field support
```
Comment (dndx) [inline CHANGELOG.md:116]:
```suggestion
- **hmac-auth**: add support for the "@request-target" pseudo-field. This ensures requests to the same target but using different request methods (such as HTTP/2) results in the same signature.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -57,9 +58,162 @@
...
+#### Hybrid Mode
+
+- Control planes don't perform health checks upon CRUD upstreams/targets events
+  [#7085](https://github.com/kong/kong/pull/7085)
+- Fixed a bug that provoked unnecessary cache flips on Data Plane
+  [#7112](https://github.com/kong/kong/pull/7112)
+- TLS keepalive request no longer can share their context
+  [#7102](https://github.com/kong/kong/pull/7102)
+- HTTP Status 405 is now handled by Kong's error handler
+  [#6933](https://github.com/kong/kong/pull/6933).
```
Comment (dndx) [inline CHANGELOG.md:155]: Those two does not seems to be Hybrid mode related changes, maybe move them under "Core".

**Comment by gszr** [comment]
Merged manually into `release/2.5.x` (which is gonna show up soon). Thanks @kikito!

**Pull request closed without merging**